Nodejs/Product/Nodejs/Debugger/NodeDebugger.cs

```diff
@@ -76,82 +76,88 @@ sealed class NodeDebugger : IDisposable {
             _fileNameMapper = new LocalFileNameMapper();
         }
 
-        public NodeDebugger(
-            string exe,
-            string script,
-            string dir,
-            string env,
-            string interpreterOptions,
-            NodeDebugOptions debugOptions,
-            ushort? debuggerPort = null,
-            bool createNodeWindow = true)
+        public NodeDebugger(Uri debuggerEndpointUri, int id)
+            : this() {
+            _debuggerEndpointUri = debuggerEndpointUri;
+            _id = id;
+            _attached = true;
+        }
+
+        public NodeDebugger(string exe, string script, string dir, string env, string interpreterOptions, NodeDebugOptions debugOptions, ushort? debuggerPort = null, bool createNodeWindow = true)
```

Comment:
This line is nearly 200 chars wide now. Please leave it wrapped as it originally was so it is readable.